pallets/utxo/src/lib.rs

```diff
@@ -286,6 +322,14 @@ pub mod pallet {
     #[allow(type_alias_bounds)]
     pub type TransactionFor<T: Config> = Transaction<T::AccountId>;
 
+    #[pallet::storage]
+    #[pallet::getter(fn token_list)]
+    pub(super) type TokenList<T> = StorageValue<_, TokenListData, ValueQuery>;
```

Comment:
Two general comments/question about `TokenList`:

1. Could this be a `StaorageMap` taking token ID to data. Should be faster when looking up token based on ID
2. Do we limit token name somehow? It is stored by all nodes and has to be in some sort of hot storage so an adversary may try to put some very big thing there.

Reply:
Agreed we should have a size limit here.

Reply:
I'm not sure we even need it that big. 5 for the ticker and 25 for the token. Going through the 100 largest coins by market cap every single one will fit in 5/25

Reply:
> I'm not sure we even need it that big. 5 for the ticker and 25 for the token. Going through the 100 largest coins by market cap every single one will fit in 5/25

Good!

Reply:
Done!